### DISCUSSION TITLE
dynamic import get the undefined

### DISCUSSION THREAD

**Discussion by lanmch** [webpack/webpack#14910] [Q&A] [answered]
recently, I update my webpack version from 4 to 5. And now I find an error after updating it.
In the past, I use _const load = await import(`@ud/biz/es/lang/locales/${locale}.js`);_ to dynamic import my module, and it works good. But now, when I run with version 5, it throw an error: _Cannot read properties of undefined (reading 'call')_[the module with console.log is undefined]. But when I change this to  _const load = await import(/* webpackChunkName: "locales-lang" */`@ud/biz/es/lang/locales/${locale}.js`);_ . It works well like past.
I really want to know, why did it not work without webpackChunkName property in webpack5? Does that have to do with the upgrade of tree shaking?
thanks a lot

**Comment by alexander-akait** [comment, marked as answer]
Please provide reproducible test repo
